docs/modules/ROOT/pages/cops_rspec.adoc

```diff
@@ -104,6 +104,7 @@ end
 
 === References
 
+* https://rubystyle.guide#any_instance_of
```

Comment:
Those are generated with the wrong domain

Reply:
I see other extensions are not using short links: https://github.com/rubocop/rubocop-rails/blob/0a902551d34f192ea3b24d7bdbbc90349d1eb9ef/config/default.yml#L63. Let me fix it real quick.

Reply:
Fixed, thanks for noticing 👍